### DISCUSSION TITLE
How to disable project filter

### DISCUSSION THREAD

**Discussion by lczerepuk** [ankitpokhrel/jira-cli#384] [Q&A] [answered]
I want to search all tasks assigned to me without "project" limit (search in all projects)  which now is impossible as I suppose. Even saved filter result which was created without project limit retrived by jira-cli add "project=XYZ" to jql.

How to ignore default project key and search in ALL projects ?

For example:

1. I have filter named e.g "assigned_to_me" with content:
`assignee = 6272xxxxxxxx and status not in ("Done")`
and I have in result issues from ALL projects

2. When I call this filter result by jira-cli with debug I can see:
```
GET /rest/api/3/search?jql=project%3D%22DEVOPS%22+AND+filter%3D%27assigned_to_me%27+ORDER+BY+created+DESC&startAt=0&maxResults=100 HTTP/1.1
```
As You see automatically jira-cli add "project" filter. As result jira-cli return only issues with this project, not for ALL projects like was filter defined. :)

**Comment by ankitpokhrel** [comment, marked as answer]
This feature is available in the latest beta release [v1.0.0-beta.2](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta.2). Details https://github.com/ankitpokhrel/jira-cli/pull/395.
